### PR TITLE
Add TypeORM icon

### LINKED ISSUE #8147: Request: TypeORM
### Brand Name

TypeORM

### Website

https://typeorm.io

### Popularity Metric

30.000 stars and 5.6 forks

### Official Resources for Icon and Color

typeorm-seeklogo com

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by davesagraf**
**Issue:** closes #8147.

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:16009]:
```suggestion
            "source": "https://github.com/typeorm/typeorm/blob/e7649d2746f907ff36b1efb600402dedd5f5a499/resources/logo_big.png",
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]
I'm happy with this treatment, though I'll need another reviewer on it given it's a manual vector job. Thanks for your work @davesagraf! 🎉

**Comment by adamrusted** [comment]
Not heard from other maintainers - so I'll merge this in. Thanks again @davesagraf!

**Comment by davesagraf** [comment]
Hi @adamrusted !

Okay, got you, thanks a lot!

**Pull request merged**
